terraform/02_project.tf

```diff
@@ -85,6 +85,22 @@ resource "google_project_service" "sourcerepo" {
   disable_dependent_services = true
 }
 
+resource "google_project_service" "gkehub" {
+  project = data.google_project.project.project_id
+
+  service = "gkehub.googleapis.com"
+
+  disable_dependent_services = true
+}
+
+resource "google_project_service" "meshconfig" {
+  project = data.google_project.project.project_id
+
+  service = "meshconfig.googleapis.com"
```

Comment:
Any particular reason why it's `meshconfig.googleapis.com` here?

I think, what you want here is `mesh.googleapis.com`? Ref:https://cloud.google.com/service-mesh/docs/unified-install/project-cluster-setup#required_apis. `mesh` is by the way enabling other APIs such as `meshconfig`, etc.

Reply:
Thanks, fixed!